src/Cabrio.java

```diff
@@ -0,0 +1,18 @@
+public class Cabrio {
+    String model;
+    String brand;
+    boolean carIsMoving;
+    boolean openRoof;
+
+    public static void openingOrClosingTheRoof(boolean carIsMoving, boolean openRoof) {
```

Comment:
This method should only take one parameter. It should also have different name -> toogleRoof